src/themes/files/light/theme-generator.json

```diff
@@ -443,9 +443,9 @@
     "h1": {
       "value": {
         "fontFamily": "{fontFamilies.main}",
+        "fontWeight": "{fontWeights.semi-bold}",
         "lineHeight": "{lineHeights.3}",
```

Comment:
What is the purpose of this?

Reply:
I simply rearranged the attribute declarations, since for all subsequent typography elements the order was the same (`fontFamily`, `fontWeight`, `lineHeight`, `fontSize` respectively). This reordering is also obviously reflected in the generated theme

Reply:
We cannot change the order of this structure since it is automatically generated by a Figma token plugin. We need to make sure with the design team whenever we change something in the theme.

Reply:
Alright, I would then communicate to the design team even this small change, together with the modifications to spacers and font size that we applied in the PR on Button